src/Dotnet.Script.Core/Scaffolder.cs

```diff
@@ -35,7 +36,13 @@ public void InitializerFolder()
                 WriteFile(pathToOmniSharpJson, omniSharpFileTemplate);
             }
 
-            CreateNewScriptFile("helloworld.csx");
+            if (Directory.GetFiles(currentDirectory, "*.csx", SearchOption.AllDirectories).Any()
```

Comment:
is the depth-scan by design? I think we don't need to scan all subfolders, just the current one?

Reply:
Agreed. Top level should be enough 👍